doc/source/ray-air/trainer.rst

```diff
@@ -212,8 +212,11 @@ You can interact with a `Result` object as follows:
     # returns the final metrics as reported
     result.metrics
 
-    # returns the contain an Exception if training failed.
+    # returns the Exception if training failed.
     result.error
 
+    # Returns a pandas dataframe of all reported results
+    results.metrics_dataframe
```

Comment:
`ResultGrid` doesn't have a `metrics_dataframe` attribute. It's a `get_dataframe()` method instead. And `get_dataframe()` only returns the last result from each trial.

Reply:
This is the `Result` class here though, right?

Reply:
Ah sorry, you meant the previous example. Fixed it

Reply:
Ah yes that’s my bad!